charts/accurate/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "accurate.fullname" . }}-manager-admin
+  labels:
+    {{- include "accurate.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "accurate.fullname" . }}-controller-manager
```

Comment:
What happens if the user chooses not to create a service account?

Reply:
refs: #11 (comment)

Reply:
What should I see?  The URL is not pointing a specific comment.

Reply:
Sorry, As mentioned in the comments below, ServiceAccount is now always created.

https://github.com/cybozu-go/accurate/pull/11#issuecomment-924537710